internal/authz/options.go

```diff
@@ -15,8 +15,9 @@ type Options struct {
 }
 
 const (
-	AllowAll = "allow-all"
-	Kessel   = "kessel"
+	AllowAll     = "allow-all"
+	Kessel       = "kessel"
```

Comment:
Here `Kessel` Refers to `relations-api`

Reply:
The reason for adding the `RelationsAPI = "kessel-relations"` const is just so that i can reference that specific value [HERE](https://github.com/project-kessel/inventory-api/blob/f8d5a3cf72b2710e91d0ef1664909722fcf10f94/internal/config/config.go#L70) instead of hardcording "kessel-relations" in that method as this is the expected name of the relations ClowdApp that shows up in appconfig. This way if it ever changes, we just change the const

Similar to how the authz.impl config has a Kessel const which i use to set the impl value to 'kessel' in
https://github.com/project-kessel/inventory-api/blob/f8d5a3cf72b2710e91d0ef1664909722fcf10f94/internal/config/config.go#L82

Reply:
I would suggest changing the `Kessel` to `RelationsAPI`. authz.Kessel  and RelationsAPI  both are referring to "kessel-relations"